### PR TITLE
Jan 15 2018 kenman345

### PR DESCRIPTION
**Pull request by kenman345**
This takes care of a lot of backlogged requests.

We can still use an image for one of the honey sites I added but otherwise, its all good. It can be merged in before we get an image if we do not receive one in the next few hours.

### CODE REVIEW

File: _data/food.yml [hunk 1]
```diff
@@ -18,6 +18,19 @@ websites:
...
         text: "Bitcoin Cash is a manual payment at this time. Contact Bees Brothers: beesbrothers@gmail.com"
 
+    - name: Crypto Mercado
+      url: https://www.cryptomercado.com/
+      img: cryptomercado.png
+      twitter: cryptomercado1
+      facebook: cryptomercadodotcom
+      region: na
+      country: US
+      city: capitalize_city_name
```
Comment (joshmcintyre) [inline _data/food.yml]: Actual city name?

File: _data/entertainment.yml [hunk 2]
```diff
@@ -30,6 +30,19 @@ websites:
...
       facebook: Crunchyroll
 
+    - name: Elven Tower
+      url: http://www.elventower.com/
+      img: elventower.png
+      twitter: elventower
+      facebook: Elven Tower
+      region: na
+      country: MX
+      city: Le
```
Comment (joshmcintyre) [inline _data/entertainment.yml]: Is Le the actual city name?

File: _data/food.yml [hunk 3]
```diff
@@ -18,6 +18,19 @@ websites:
...
+      bch: yes
+      btc: yes
+      othercrypto: yes
+      doc:
```
Comment (joshmcintyre) [inline _data/food.yml]: Add doc link or remove this empty field.

File: _data/retail.yml [hunk 4]
```diff
@@ -589,6 +611,17 @@ websites:
       exceptions:
         text: "BCH payments are accepted but only manually at this time."
 
+    - name: Sutton Honey
+      url: http://suttonhoney.com
+      img:
```
Comment (joshmcintyre) [inline _data/retail.yml]: Add img link or remove empty field.

File: _data/retail.yml [hunk 5]
```diff
@@ -589,6 +611,17 @@ websites:
...
+    - name: Sutton Honey
+      url: http://suttonhoney.com
+      img: 
+      region: na
+      country: CA
+      city: Sutton
+      bch: yes
+      btc: yes
+      othercrypto: yes
+      doc:
```
Comment (joshmcintyre) [inline _data/retail.yml]: Same here, doc link or remove empty field.

File: _data/wallets.yml [hunk 6]
```diff
@@ -132,6 +132,19 @@ websites:
...
+      img: luno.png
+      twitter: lunomoney
+      facebook: luno
+      region: af
+      country: za
+      city: Cape Town
+      bch: no
+      btc: yes
+      othercrypto: yes
+      doc:
```
Comment (joshmcintyre) [inline _data/wallets.yml]: See above.

File: _data/food.yml (hunk 1 above)
Comment (kenman345) [reply _data/food.yml]: oops, Will remove

File: _data/food.yml (hunk 3 above)
Comment (kenman345) [reply _data/food.yml]: tags can be empty, not a huge deal

**Comment by kenman345** [comment]
@joshmcintyre  I made some adjustments based on your feedback. I think these changes make sense. Good catch on some of those.

File: _data/entertainment.yml (hunk 2 above)
Comment (kenman345) [reply _data/entertainment.yml]: yea, not sure about that one, but we're not using that tag yet

**Comment by kenman345** [comment]
Made some changes @joshmcintyre 

Take a look when you can.

**Pull request merged**